tests/virt/node/descheduler/utils.py

```diff
@@ -261,7 +233,9 @@ def deploy_vms(
         vm.delete()
 
     for vm in vms:
-        vm.wait_deleted()
+        # Due to the bug - VM may hang in terminating state, need to remove the finalizer from VMI
+        if not vm.wait_deleted() and is_jira_open(jira_id="CNV-67515"):
```

Comment:
`is_jira_open(jira_id="CNV-67515")`
make this cached func or fixture. it's being called for every not deleted VM